tests/multi_ptr/multi_ptr_access_members.h

```diff
@@ -0,0 +1,156 @@
+
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides code for multi_ptr access members
+//
+*******************************************************************************/
+#ifndef __SYCLCTS_TESTS_MULTI_PTR_ACCESS_MEMBERS_OPS_H
+#define __SYCLCTS_TESTS_MULTI_PTR_ACCESS_MEMBERS_OPS_H
+
+#include "../common/common.h"
+
+#include "../common/section_name_builder.h"
+#include "multi_ptr_common.h"
+
+namespace multi_ptr_access_members {
+
+/**
+ * @brief Provides verification for multi_ptr access operators that they returns
+ *        correct data types
+ * @tparam T Current data type
+ * @tparam AddrSpaceT sycl::access::address_space enumeration's field
+ * @tparam IsDecorated sycl::access::decorated enumeration's field
+ * @param type_name Current data type string representation
+ * @param address_space_name Current sycl::access::address_space string
+ *        representation
+ * @param is_decorated_name Current sycl::access::decorated string
+ *        representation
```

Comment:
I think we should move this to the `void operator()` definition

Reply:
this comment applied [here](https://github.com/KhronosGroup/SYCL-CTS/pull/346/commits/4bcaf862cf82c80d669077c2eb2ffc920474524a#diff-7c3b5239659a41d2cc6341067d6094f03525239f4a9380fce5a1056b6bfddc62R33-R39)